examples/android/app/build.gradle

```diff
@@ -1,34 +1,67 @@
 apply plugin: 'com.android.application'
+apply plugin: 'com.google.protobuf'
 
 android {
-    compileSdkVersion 21
-    buildToolsVersion "21.1.2"
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
 
     defaultConfig {
-        applicationId "io.grpc.helloworldexample"
-        minSdkVersion 7
-        targetSdkVersion 21
+        applicationId "io.grpc.android.helloworldexample"
+        minSdkVersion 9
+        targetSdkVersion 22
         versionCode 1
         versionName "1.0"
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    lintOptions {
+        disable 'InvalidPackage', 'HardcodedText'
+    }
+}
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:3.0.0-beta-1'
+    }
+    plugins {
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java:1.0.0-SNAPSHOT'
```

Comment:
Either that, or we can make this based on the latest release... Which might be better since it isn't integrated into our build.

@zhangkun83, do you have an opinion?